octree/include/pcl/octree/octree_pointcloud.h

```diff
@@ -428,7 +428,7 @@ class OctreePointCloud : public OctreeT {
    * \a setInputCloud to be added
    */
   virtual void
-  addPointIdx(const int point_idx_arg);
+  addPointIdx(index_t point_idx_arg);
```

Comment:
uindex_t?

Reply:
Should const stay? (multiple occurencies)

Reply:
`const` on a by-value parameter doesn't have any influence on the API. If only makes sense in a definition, not declaration (since it doesn't impose any constraints)